### ISSUE TITLE
`Element type is invalid` while using Chakra UI and Next.js

### ISSUE THREAD

**Issue by Helloyunho** [oven-sh/bun#950]
### Version

0.1.5

### Platform

Darwin (redacted).local 21.6.0 Darwin Kernel Version 21.6.0: Sat Jun 18 17:07:22 PDT 2022; root:xnu-8020.140.41~1/RELEASE_ARM64_T6000 arm64

### What steps will reproduce the bug?

1. Create a simple Next.js project
2. Add necessary dependencies `@chakra-ui/react @emotion/react@^11 @emotion/styled@^11 framer-motion@^6`
   - From [official Chakra UI setup guide](https://chakra-ui.com/getting-started/nextjs-guide)
3. Setup `pages/_app.tsx` and `pages/_document.tsx`
```tsx
// pages/_app.tsx
import { ChakraProvider } from '@chakra-ui/react'

const MyApp = ({ Component, pageProps }) => {
  return (
    <ChakraProvider>
      <Component {...pageProps} />
    </ChakraProvider>
  )
}

export default MyApp
```
```tsx
// pages/_document.tsx

import NextDocument, { Html, Head, Main, NextScript } from 'next/document'

export default class Document extends NextDocument {
  render() {
    return (
      <Html lang='en'>
        <Head />
        <body>
          <Main />
          <NextScript />
        </body>
      </Html>
    )
  }
}
```
  - Also from [official Chakra UI setup guide](https://chakra-ui.com/getting-started/nextjs-guide)
  - Slightly modified to use ES6 syntax
4. Modify `pages/index.tsx` to use Chakra UI's components
```tsx
// pages/index.tsx

import React from 'react'
import { Button } from '@chakra-ui/react'

const Index = () => {
  return (
    <>
      <Button>Hello</Button>
    </>
  )
}

export default Index
```
5. Open dev server and go to it `http://localhost:3000`

### How often does it reproduce? Is there a required condition?

Every time when I enter(refresh) the page.

### What is the expected behavior?

It should not throw any errors and render without the error message.
- I removed the error message in the screenshot below
<img width="2088" alt="A picture of page without an error message" src="https://user-images.githubusercontent.com/10192625/182158532-606785fc-0142-4a06-867b-4cb69f9a410e.png">

### What do you see instead?

Error below will appear (sometimes different error but this element error keeps occurring)
```
Element type is invalid:
expected a string (for built-in components) or a class/function (for composite components) but got: undefined.
You likely forgot to export your component from the file it's defined in,
or you might have mixed up default and named imports.
```

Fun fact: Even though the error occurred, it has no problem rendering the page.
![Picture of perfectly working page even though the error has occurred.](https://user-images.githubusercontent.com/10192625/182158391-af97beac-b19a-4ef9-9cb8-1570a3538808.png)

### Additional information

This is my first issue posting in bun and you may want more information about this issue. Feel free to ask me anytime and as you probably know I'm on Discord too so you can also use that.

**Comment by Helloyunho** [comment]
I tried the same code with the official Next.js CLI and it works flawlessly without an issue.

**Comment by Helloyunho** [comment]
Seems like it's fixed from bun 0.1.7, probably #945 fixed it
